<ui version="4.0" >
 <class>TurbulenceAdvancedOptionsDialogForm</class>
 <widget class="QDialog" name="TurbulenceAdvancedOptionsDialogForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string/>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelScales" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IDEUCH</string>
            </property>
            <property name="text" >
             <string>Wall function type</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxScales" >
            <property name="minimumSize" >
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Code_Saturne key word: IDEUCH</string>
            </property>
            <item>
             <property name="text" >
              <string>One scale model</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Two scale model</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Scalable wall function</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelGravity" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IGRAKE / IGRARI</string>
            </property>
            <property name="text" >
             <string>Gravity terms in the turbulence equations</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxGravity" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IGRAKE / IGRARI</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TurbulenceAdvancedOptionsDialogForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TurbulenceAdvancedOptionsDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
